apis/core/v1alpha1/identifiers.go

```diff
@@ -21,6 +21,9 @@ type AWSIdentifiers struct {
 	// NameOrId is a user-supplied string identifier for the resource. It may
 	// or may not be globally unique, depending on the type of resource.
 	NameOrID *string `json:"nameOrID,omitempty"`
+	// AdditionalKeys represents any additional arbitrary identifiers used when
+	// describing the target resource.
+	AdditionalKeys map[string]*string `json:"additionalKeys,omitempty"`
```

Comment:
Any reason this is `map[string]*string` instead of `map[string]string`?

Reply:
I was using the other `*string` fields as reference. Is there a hard and fast rule to when we should use `*string` over `string`?

Reply:
The use of `map[string]*string` is really an aws-sdk-go thing. Since this isn't an aws-sdk-go shape field, I just thought it would make more sense to use a standard `map[string]string`. Pointer types generally signal optional fields (except for aws-sdk-go, which makes everything a pointer type, unfortunately).

Reply:
Fair enough. I've updated all unnecessary `*string` to `string` types for the adopted resource APIs.